### ISSUE TITLE
Door Status bug in 1.0.8

### ISSUE THREAD

**Issue by itchannel** [itchannel/fordpass-ha#53]
Status check is not working correctly. All doors showing Closed but main Status is open

```          for key, value in self.coordinator.data[self.sensor].items():
                    if (value["value"] != "Closed") or (value["value"] != "Invalid"):
                        return "Open"
                return "Closed"```

**Comment by Afterdark1st** [comment]
I Have the same problem

**Comment by itchannel** [comment]
Fixed in the latest release #54
